src/test/java/org/mule/extension/ftp/FtpCopyTestCase.java

```diff
@@ -9,19 +9,22 @@
 import static java.lang.String.format;
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.junit.Assert.assertThat;
-import static org.mule.extension.file.common.api.util.UriUtils.createUri;
-import static org.mule.test.extension.file.common.api.FileTestHarness.HELLO_WORLD;
-import static org.mule.extension.file.common.api.exceptions.FileError.FILE_ALREADY_EXISTS;
-import static org.mule.extension.file.common.api.exceptions.FileError.ILLEGAL_PATH;
+import static org.mule.extension.ftp.internal.util.UriUtils.createUri;
+import static org.mule.extension.ftp.api.FileTestHarness.HELLO_WORLD;
+import static org.mule.extension.ftp.internal.error.FileError.FILE_ALREADY_EXISTS;
+import static org.mule.extension.ftp.internal.error.FileError.ILLEGAL_PATH;
 import static org.mule.extension.ftp.AllureConstants.FtpFeature.FTP_EXTENSION;
 import static org.mule.extension.ftp.internal.FtpUtils.normalizePath;
-import org.mule.extension.file.common.api.exceptions.FileAlreadyExistsException;
-import org.mule.extension.file.common.api.exceptions.IllegalPathException;
+
+import org.junit.Ignore;
+import org.mule.extension.ftp.internal.exception.FileAlreadyExistsException;
+import org.mule.extension.ftp.internal.exception.IllegalPathException;
 
 import io.qameta.allure.Feature;
 import org.junit.Test;
 
 @Feature(FTP_EXTENSION)
+@Ignore
```

Comment:
Why is this test ignored?

Reply:
@elcoya 
we are getting issue with these junits after we have upgraded parent pom version (mule-core-modules-parent) from **1.1.9 to 1.3.2**

**We have also written corresponding munits for the junits ignored:**
https://github.com/mulesoft/mule-ftp-connector/pull/426/commits/c38631d05efc0a23175e166e49aaa71a5f6e0ddb
https://github.com/mulesoft/mule-ftp-connector/pull/426/commits/4227899be936d81337a42db1cccd7cced7a9ae2b

**we are not sure how these junits can be fixed, please let us know if you know a way to fix these or can we remove these junits since we have corresponding munits for it?**

cc @dipeshkumardutta-sfemu

Reply:
Seems like there are multiple issues with the docker images being used. Will have to troubleshoot and workout to solve them.